app/src/main/java/chat/rocket/android/layouthelper/ExtRealmModelListAdapter.java

```diff
@@ -37,6 +41,35 @@ protected void notifyRealmModelItemChanged(int position) {
     notifyItemChanged(position + 1);
   }
 
+  public void setLayoutManager(RecyclerView.LayoutManager layoutManager) {
+    this.layoutManager = layoutManager;
+  }
+
+  protected ListUpdateCallback listUpdateCallback = new ListUpdateCallback() {
+    @Override
+    public void onInserted(int position, int count) {
+      notifyItemRangeInserted(position + 1, count);
+      if (layoutManager != null) {
+        layoutManager.scrollToPosition(0);
+      }
```

Comment:
I agree with avoiding the hard Realm dependency.
I actually prefer the method with Rx like your suggestion 😅 .

Anyway, shall we separate the stuff other than related to DiffUtils here?
https://github.com/RocketChat/Rocket.Chat.Android/pull/146/commits/ae5211eb3b4c64852bc555e3b28ff1d2803ef849
I created a commit to remove it. Could you provide a new PR with reverting the commit?